diff --git a/configure.ac b/configure.ac
index 7ab65a876..688fd4ee5 100644
--- a/configure.ac
+++ b/configure.ac
@@ -101,7 +101,11 @@ echo "Checking packages required for SWIG developers."
 echo "Note : None of the following packages are required for users to compile and install SWIG from the distributed tarball"
 echo ""
 
-AC_CHECK_PROGS([BISON], [bison], [$MISSING bison])
+if test -z "$BISON"; then
+  AC_CHECK_PROGS([BISON], [bison], [$MISSING bison])
+else
+  AC_MSG_NOTICE([Using bison from environment variable BISON... ${BISON}])
+fi
 
 echo ""
 echo "Checking for installed target languages and other information in order to compile and run"
