--- configure.ac.orig	2024-04-01 22:26:44.052141496 +0200
+++ configure.ac	2024-04-01 22:27:30.347837629 +0200
@@ -2863,7 +2863,9 @@
 # source directory, and provides a rule for updating the Makefile from
 # its original source.
 AC_CONFIG_COMMANDS([Examples],[
-  if test "x${srcdir}" != "x." ; then
+  CONFIG_SRC_DIR=`cd ${srcdir} && pwd`
+  CONFIG_TARGET_DIR=`pwd`
+  if test "x${CONFIG_SRC_DIR}" != "x${CONFIG_TARGET_DIR}" ; then
     AC_MSG_NOTICE([generating Examples build tree])
     for mkfile in `cd ${srcdir} && find Examples/ -type f -name Makefile`; do
       dir=`dirname ${mkfile}`
