diff --git a/configure.ac b/configure.ac
index 1bacc3847..a2b31fa99 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2924,7 +2924,9 @@ AC_CONFIG_FILES([CCache/ccache_swig_config.h])
 # source directory, and provides a rule for updating the Makefile from
 # its original source.
 AC_CONFIG_COMMANDS([Examples],[
-  if test "x${srcdir}" != "x." ; then
+  CONFIG_SRC_DIR=`cd ${srcdir} && pwd`
+  CONFIG_TARGET_DIR=`pwd`
+  if test "x${CONFIG_SRC_DIR}" != "x${CONFIG_TARGET_DIR}" ; then
     AC_MSG_NOTICE([generating Examples build tree])
     for mkfile in `cd ${srcdir} && find Examples/ -type f -name Makefile`; do
       dir=`dirname ${mkfile}`
